maintainers/maintainer-list.nix

```diff
@@ -7523,16 +7523,6 @@
     githubId = 29855073;
     name = "Michael Colicchia";
   };
-  matdsoupe = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: update NAME`` and sort it alphabetically.